Cargo.toml

```diff
@@ -24,7 +24,9 @@ keywords = [
 features = ["ci-check", "f16", "cudnn"]
 
 [features]
-default = ["std", "driver", "nvrtc", "cublas", "curand"]
+default = ["std", "driver", "nvrtc", "cublas", "curand", "cuda_version_11_8"]
+cuda_version_11_8 = []
+cuda_version_12_2 = []
```

Comment:
These could probably be set automatically in build.rs based on whatever CUDA is on the machine